DQMOffline/Trigger/python/MssmHbbMonitor_cfi.py

```diff
@@ -29,8 +50,29 @@
     triggerobjbtag = cms.string("hltBTagCalo30x8CSVp0p92SingleWithMatching"),
```

Comment:
Could be done in principle. But this way it is better for controlling the parameters values for each module call.